packages/stylex/package.json

```diff
@@ -34,13 +24,12 @@
   },
   "license": "MIT",
   "scripts": {
-    "prebuild": "gen-types -i src/ -o lib/",
-    "build:cjs": "cross-env BABEL_ENV=cjs babel src/ --out-dir lib/ --copy-files",
-    "build:esm": "cross-env BABEL_ENV=esm babel src/ --out-dir lib/es --out-file-extension .mjs",
-    "postbuild:esm": "rollup -c ./rollup.config.mjs",
+    "prebuild": "rimraf lib && gen-types -i src/ -o lib/cjs && cp -r lib/cjs lib/es",
```

Comment:
This cleans out the build directory before every build, to prevent us potentially publishing to npm with artifacts from past versions that might be lying around on your local filesystem. It also copies the types across cjs/es dirs for when using flow.